### PR TITLE
styled the popup message and added animation

### PR DESCRIPTION
**Pull request by phillnab**
## What It Does
Adds style to the popup message and placement of it on the screen.

Also adds a transition animation to have it fade in/up and out/down.

## How To Test

You can adjust the animation speeds and popup ares in the .message-box and .show-message classes.

## Checklist

Under penalty of public shaming, I avow that I:

- [ ] Reviewed the diff to look for typos, whitespace errors, and console.log() statements
- [ ] Verified that there are no compiler or linter errors or warnings
- [ ] Verified the build in production(optimized) mode
- [ ] Updated the docs, if necessary
- [ ] Included the appropriate labels
- [ ] Referenced applicable issues (i.e. Closes #XXXX, Fixes #XXXX)

Browsers tested:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] Internet Explorer

## Dependencies

- [ ] any other PRs or issues that should be resolved/merged before this is merged

### CODE REVIEW

File: src/user-list/user-list.html [hunk 1]
```diff
@@ -147,8 +164,8 @@ <h2>There are currently no users in the database. Please create one.</h2>
       popupMessage(action) {
         let animationTime = 3000;
         let messageBox = this.$.popupBox;
-        messageBox.textContent = action + 'Successful';
-        messageBox.className = 'message-box show-message';
+        messageBox.textContent = action + ' Successful';
+        messageBox.className = 'message-box show-message ' + action;
```
Comment (Jropp) [inline src/user-list/user-list.html]: Was there a particular reason for adding the action as a class name here?

Comment (collincahill) [reply src/user-list/user-list.html]: Not truly the best way to accomplish this, but to have effective and granular control over colors and messages you'd have to refactor a good amount of the popup code.

Comment (phillnab) [reply src/user-list/user-list.html]: We were trying to find a way to have different background colors depending on the message.  Like failed red or success green.  And I believe this helps make that work once we get to that here soon. Sorry I forgot about it and by we I mean Collin and I.

Comment (Jropp) [reply src/user-list/user-list.html]: Sounds great! I just wanted to make sure it wasn’t an error.

**Pull request merged**